roles/container-engine/docker/tasks/main.yml

```diff
@@ -11,6 +11,13 @@
   set_fact:
     is_ostree: "{{ ostree.stat.exists }}"
 
+- name: set docker_version for openEuler
+  set_fact:
+    docker_version: '19.03'
+  when: ansible_distribution == "openEuler"
+  tags:
+    - facts
+
```

Comment:
Thanks @ErikJiang 

I guest the code may be unnecessary, the docker version can also be set at the config YAML.
The openEuler can support the higher docker version.

Reply:
When deploying a cluster in openEuler 22.03 LTS with the default docker-ce version 20.10, the following error occurs.
![image](https://user-images.githubusercontent.com/10629406/202345311-3a5ac69a-5fcf-42e4-8c44-a9c7ccf8235b.png)

Reply:
Thanks @ErikJiang 

/lgtm